MyDataHelpsKit/Model/DeviceData.swift

```diff
@@ -61,23 +71,31 @@ public struct DeviceDataResultPage: PagedResult, Decodable {
     /// An ID to be used with subsequent `DeviceDataQuery` requests. Results from queries using this ID as the `pageID` parameter will show the next page of results. `nil` if there isn't a next page.
     public let nextPageID: String?
 }
-
-/// Specifies the source framework for the device data.
+    
+/// Device data is grouped into namespaces, which represent the source frameworks that generate the data. There is also a separate `project` namespace, where projects can persist their own data. The static members of DeviceDataNamespace identify all supported namespace values.
 public struct DeviceDataNamespace: RawRepresentable, Equatable, Decodable {
     public typealias RawValue = String
     
-    /// Data imported from a linked Apple Health account.
-    public static let appleHealth = DeviceDataNamespace(rawValue: "AppleHealth")
+    /// Data persisted by the Persist Device Data Points operation.
```

Comment:
Suggest:  "Project-specific device data."
There may be other operations in the future that could alter project device data, or a GUI to edit it in RKS itself.